ui/src/app/pages/requests/requests.component.ts

```diff
@@ -55,6 +69,48 @@ export class RequestsComponent {
                 new TableHeader('Group Requested', '', 'function', (element) => {
                     return this.groupUtils.getGroup(element.object).name;
                 }),
+                new TableHeader('Join / Remove', '', 'function', (element) => {
```

Comment:
This TableHeader is duplicated across RequestsComponent and PendingApprovalComponent. Would it make sense to extract it into an object that each component uses instead?

Reply:
TableHeader registers to a column on the UI. might be an over kill to take out the TableHeader? I am not sure whether I get the comment right?

Reply:
I just noticed that there is a shared TableHeader across a few views in the app and wondered how easily it could be extracted (so changes to the table header would be reflected across all tables having the same header). Considering it is a subset of the entire table and that tables are different, it could end up being overkill to extract it as you said.